tests/experimental/metric_repository/test_column_descriptive_metrics_metric_retriever_integration.py

```diff
@@ -189,6 +189,20 @@ def test_get_metrics(
             value=0,
             exception=None,
         ),
+        ColumnMetric[str](
+            batch_id=batch_id,
+            metric_name="column.min",
+            value=Timestamp("2020-01-01 00:00:00"),
+            exception=None,
+            column="datetime",
+        ),
+        ColumnMetric[str](
+            batch_id=batch_id,
+            metric_name="column.max",
+            value=Timestamp("2020-01-03 00:00:00"),
+            exception=None,
+            column="datetime",
+        ),
```

Comment:
Note that timestamp min/max metrics are not always returned as strings (but are for Snowflake). We will need to address when more datasources are supported.